.travis.yml

```diff
@@ -3,7 +3,6 @@ language: php
 php:
   - 5.6
   - 7.0
-  - hhvm
```

Comment:
in the other projects, i added this to the `matrix` section:

```
- php: hhvm
  dist: trusty
```

that worked fine so far.